packages/expo-updates/ios/EXUpdates/Database/EXUpdatesDatabase.m

```diff
@@ -449,6 +452,97 @@ - (nullable EXUpdatesAsset *)assetWithKey:(NSString *)key error:(NSError ** _Nul
   }
 }
 
+# pragma mark - json data
```

Comment:
wow, a lot of raw sql. Is there just no need/desire for an ORM?

Reply:
TBH this DB started out simpler, and since SQLite is natively supported in obj-c and I'm comfortable writing SQL, I decided not to add any extra dependencies and just interact with SQLite directly. The database has grown, however, so I'm open to adding an ORM in the future! Android's `Room` is quite nice to use.

Reply:
That makes sense!

Reply:
SQL is often easier than an ORM if the goal is just to query data from SQL. The benefit of something like Entity is not to avoid SQL, it's to handle caching, privacy policies, transactions in a way that works together.